src/routes/Profile/data/getSeasonBadges/getSeasonBadges.ts

```diff
@@ -2,40 +2,30 @@ import { Season } from '../../../../hooks/season/types';
 import { BADGE_TYPE_BADGE_VARIANT_MAP, SEASON_BADGE_VARIANTS } from './mappers';
 import { BadgeVariant, GetProfileBadgesResponse } from './types';
 import { getDefaultResponse } from './helpers';
-import { Signer } from 'ethers';
 import { CommunitySBT } from '@voltz-protocol/v1-sdk';
 
 export async function getSeasonBadges({
   userId,
-  seasonId,
-  signer,
+  season,
+  sbt
 }: {
   userId: string;
-  seasonId: Season['id'];
-  signer: Signer;
+  season: Season;
+  sbt: CommunitySBT
 }): Promise<GetProfileBadgesResponse> {
   if (!process.env.REACT_APP_SUBGRAPH_BADGES_URL) {
-    return getDefaultResponse(seasonId);
```

Comment:
next time around you can just do `const seasonId = season.id` and don't have to change 5 lines below :)